Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM python:3.8
+FROM python:3.10
 
 WORKDIR /app
 
 # Install specific optimade and aiida-core versions
-ARG OPTIMADE_TOOLS_VERSION=0.16.12
-ARG AIIDA_VERSION=1.6.8
+ARG OPTIMADE_TOOLS_VERSION=0.16.10
```

Comment:
This should be updated to the latest version by rebasing on or merging in `master`.

Reply:
I'll do this :)